pkg/server/proto/server.proto

```diff
@@ -496,10 +496,42 @@ service Waypoint {
   // Waypoint project to the database
   rpc CreateProjectFromTemplate(CreateProjectFromTemplateRequest) returns (CreateProjectFromTemplateResponse);
 
+  // Deprecated in favor of GetTFRunStatus
   // GetTFCRunStatus is in BETA. It returns the status of a TFC run, which was
   // run to render a project template.
   rpc GetTFCRunStatus(GetTFCRunStatusRequest) returns (GetTFCRunStatusResponse);
 
+  // CreateAddOnDefinition creates a new Add On definition
+  rpc CreateAddOnDefinition(CreateAddOnDefinitionRequest) returns (CreateAddOnDefinitionResponse);
+
+  // GetAddOnDefinition returns an Add On definition, referenced by name or ID
+  rpc GetAddOnDefinition(GetAddOnDefinitionRequest) returns (GetAddOnDefinitionResponse);
+
+  // ListAddOnDefinitions returns all existing Add On definitions
+  rpc ListAddOnDefinitions(ListAddOnDefinitionsRequest) returns (ListAddOnDefinitionsResponse);
+
+  // UpdateAddOnDefinition updates an existing Add On definition
+  rpc UpdateAddOnDefinition(UpdateAddOnDefinitionRequest) returns (UpdateAddOnDefinitionResponse);
+
+  // DeleteAddOnDefinition deletes an existing Add On definition
+  rpc DeleteAddOnDefinition(DeleteAddOnDefinitionRequest) returns (google.protobuf.Empty);
+
+  // CreateAddOn adds an Add On to a project from an Add On definition
+  rpc CreateAddOn(CreateAddOnRequest) returns (CreateAddOnResponse);
+
+  // DestroyAddOn destroys an existing Add On
```

Comment:
Is there a distinction we commonly use to know when to use `Destroy` vs. `Delete` verbs?

Reply:
I don't think we have a standard yet, but I like the idea that "destroy" means active resources running need to be destroyed where "delete" is a simple database delete from the Waypoint DB. That's how I've been thinking about it, at least!

Reply:
+1 to what @briancain said!